alembic/versions/9152eefe06e0_create_initial_application_table.py

```diff
@@ -0,0 +1,36 @@
+"""Create initial application table
+
+Revision ID: 9152eefe06e0
+Revises:
+Create Date: 2023-08-09 12:54:19.543072
+
+"""
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import JSON, UUID
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "9152eefe06e0"
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.create_table(
+        "application",
+        sa.Column("id", sa.Integer, primary_key=True),
```

Comment:
any reason this isn't a UUID? given we (potentialy) want to use this across services.. or at the very least a bigint

Reply:
Other than i was just testing this works for us, no. Have changed to UUID